tests/upgrades/test_host.py

```diff
@@ -219,6 +195,6 @@ def test_post_create_gce_cr_and_host(
         ).create()
         request.addfinalizer(pre_upgrade_host.delete)
         request.addfinalizer(host.delete)
-        assert host.name == f"{self.hostname}-{domain}"
+        assert host.name == f"{self.hostname.lower()}.{domain.name}"
```

Comment:
@omkarkhatavkar I needed to change this one somehow the name was getting camel cased

Reply:
okay make sense.